arquillian-environment-reporter/arquillian-environment-reporter-impl/src/main/java/org/arquillian/environment/reporter/impl/ArquillianEnvironmentKeys.java

```diff
@@ -0,0 +1,19 @@
+package org.arquillian.environment.reporter.impl;
+
+import org.arquillian.reporter.api.model.AbstractStringKey;
+import org.arquillian.reporter.api.model.StringKey;
+
+public class ArquillianEnvironmentKeys extends AbstractStringKey {
```

Comment:
Following convention of other extensions of recorder. Also notice that this class contains keys and not key.